packages/_shared/src/FieldConnector.ts

```diff
@@ -71,34 +69,30 @@ export class FieldConnector<ValueType> extends React.Component<
   unsubscribeDisabled: Function | null = null;
   unsubscribeValue: Function | null = null;
 
-  setValue = throttle(
+  setValue = async (value: ValueType | Nullable) => {
+    if (this.props.isEmptyValue(value === undefined ? null : value)) {
```

Comment:
```suggestion
    if (this.props.isEmptyValue(value ?? null)) {
```

What do you think about doing this ☝️ ?

Reply:
Normally I am the person mentioning these fixes in reviews 🤔